src/master-playlist-controller.js

```diff
@@ -599,25 +642,6 @@ export class MasterPlaylistController extends videojs.EventTarget {
     });
 
     this.mainSegmentLoader_.on('earlyabort', (event) => {
-      if (this.experimentalBufferBasedABR) {
```

Comment:
Should we return here for `this.experimentalBufferBasedABR` so that we don't use earlyabort?

Reply:
I turned off triggering earlyabort in the segment loader, so, theoretically, we should never get here if `experimentalBufferBasedABR` is set.